quiche/src/h3/mod.rs

```diff
@@ -1670,6 +1793,25 @@ impl Connection {
 
         conn.stream_send(stream_id, grease_payload, false)?;
 
+        trace!(
+            "{} tx frm GREASE stream={} len=18",
```

Comment:
Probably better to use `grease_payload.len()` rather than hardcode the value.

Reply:
agree